web/pages/api/auth/signin/index.js

```diff
@@ -9,21 +9,16 @@ const handler = nextConnect().post(async (req, res) => {
     res.status(400).json({ error: "Email or Password cannot be blank" });
 
   try {
-    var results = await query(
+    var [results] = await query(
       `
               SELECT password, id FROM users WHERE email = ?
             `,
       [email]
-    );
-
-    if (results[0].length < 1)
-      res.status(401).json({ error: "Email not found" });
-    else results = JSON.parse(JSON.stringify(results[0][0])); // this is needed to parse the RowPacket that mysql returns
-
-    const { password: hashedPW, id } = results;
+    );    
+    if (results.length < 1) res.status(401).json({ error: "Email not found" });
```

Comment:
Good for debug! We need to note later to change these messages to not giveaway these details.

It's standard security to reject all logins with the same message, so that an 'attacker' doesn't get free information like "hey this email exists, but I got the password wrong."

Reply:
Never thought about it that way, makes sense